usr/share/rear/prep/USB/default/060_set_usb_device.sh

```diff
@@ -30,8 +30,8 @@ test "$USB_PREFIX" || USB_PREFIX="rear/$HOSTNAME/$(date +%Y%m%d.%H%M)"
 
 ### Change NETFS_PREFIX to USB_PREFIX if our backup URL is on USB
 if [[ "$BACKUP_URL" ]] ; then
-    local scheme=$(url_scheme $BACKUP_URL)
-    case $scheme in
+    local scheme="$( url_scheme "$BACKUP_URL") "
```

Comment:
This looks like an added whitespace error - scheme will have a trailing space and the code will malfunction.

Reply:
well spotted!